src/main/kotlin/com/criptext/mail/scenes/mailbox/MailboxSceneController.kt

```diff
@@ -1207,84 +1231,71 @@ class MailboxSceneController(private val scene: MailboxScene,
                     }
                 }
             }
-            generalDataSource.submitRequest(GeneralRequest.TotalUnreadEmails(model.selectedLabel.text))
-            scene.updateToolbarTitle(toolbarTitle)
```

Comment:
some of this lines were removed but not added in other lines. E.g. `feedController.reloadFeeds()` and `scene.showNotification()`